07/hello.go

```diff
@@ -0,0 +1,9 @@
+package hello
+
+func ReturnGreeting(s string) string {
+	if s == "Hi there" {
+		return "Hi there yourself!"
+	} else {
```

Comment:
There's a nice refactoring we often make in Go; any time we have something like:

```go
if X {
	return something
} else {
	return something else
}
```

we can drop `else` and un-indent the other branch:

```go
if X {
	return something
}
return something else
```

Clearly, if we're still executing by the time we reach `return something else`, then X was false, so `else` is unnecessary!